conf/application.conf

```diff
@@ -120,3 +120,5 @@ contact-frontend {
 mongodb {
   uri = "mongodb://localhost:27017/hello-world-upscan"
 }
+logger.uk.gov = WARN
+logger.application = WARN
```

Comment:
these defaults can be left in `application-json-logger.xml` (development defaults are in `logback.xml`)

Reply:
Done